getsession/internal/config/config.go

```diff
@@ -0,0 +1,47 @@
+package config
+
+import (
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/joho/godotenv"
+)
+
+type Config struct {
+	TelegramApiId   int
+	TelegramApiHash string
+	Phone           string
```

Comment:
I'd rather pass here session file (in env it could be passed in base64 format), please check out previous comment for details